wix-bookings/guides/Terminology.md

```diff
@@ -1,58 +1,57 @@
 # Wix Bookings Terminology
```

Comment:
Maybe consider adding more terms, like policy, eCommerce... maybe some "cancellation fee"-related terms? 
I think these things were added after glossary was written. I wonder what other new features popped up that we can add to the list... Obviously we can do this over time... not essential for now.

Reply:
Yes, we should find a system to add terms whenever they're introduced in any API. Policy is already there, and link to eCom from the checkout description. I hope I'll remember to add cancellation fee once it has been released in the APIs.